pkg/kotsutil/troubleshoot.go

```diff
@@ -50,3 +51,26 @@ func LoadTSKindsFromPath(dir string) (*troubleshootloader.TroubleshootKinds, err
 	}
 	return tsKinds, nil
 }
+
+func Dedup[T any](objs []T) []T {
```

Comment:
Reconsider the implementation of `Dedup`. Instead of using JSON marshaling to check for duplicates, consider if there's a simpler way that takes advantage of the properties of the objects you're working with (e.g., unique fields or a specific interface method). This will improve the performance and clarity of the code.

Reply:
Thank you! I will close this PR and update the change in https://github.com/replicatedhq/kots/pull/4958